README.md

```diff
@@ -0,0 +1,35 @@
+# com.jetbrains.GoLand
+
```

Comment:
Not sure if this adds any value to what's already on the website & docs?

Reply:
I'm just copying what has been done for other JetBrains products for flathub.

Reply:
It should be removed. Each command is covered by regular flatpak documentation.

Reply:
Ok, I removed all the parts related to flatpak.

Reply:
It's even more unnecessary now. No user is going to read this, this is what appdata is for.

Reply:
so better just delete the readme ?

Reply:
Yes.

Reply:
Ok removed :+1: